packages/cli-vector/src/cli/cli.analyse.ts

```diff
@@ -0,0 +1,192 @@
+import sq from 'node:sqlite';
+
+import { fsa, Url } from '@basemaps/shared';
+import { CliInfo } from '@basemaps/shared/build/cli/info.js';
+import { getLogger, logArguments } from '@basemaps/shared/build/cli/log.js';
+import { VectorTile } from '@mapbox/vector-tile';
+import { command, option, positional } from 'cmd-ts';
+import { readFileSync } from 'fs';
+import Mustache from 'mustache';
+import sizeof from 'object-sizeof';
+import { basename } from 'path';
+import Protobuf from 'pbf';
+import { gunzip } from 'zlib';
+
+interface TileInfo {
+  x: number;
+  y: number;
+  z: number;
+  max: number;
+  size?: string;
+}
+
+interface Distribution {
+  distribution: string;
+  tiles: number;
+  percentage: string;
+}
+
+interface LayerSum {
+  features: number;
+  totalGeometry: number;
+  totalAttributes: number;
+}
+
+interface LayerInfo {
+  name: string;
+  features: number;
+  totalGeometry: string;
+  totalAttributes: string;
+  totalSize: string;
+}
+
+interface AnalysisData {
+  zoom: number;
+  maxTile: TileInfo;
+  distributions: Distribution[];
+  layers: LayerInfo[];
+}
+
+function distribution(size: number): string {
+  if (size <= 10000) return '0-10KB';
+  if (size <= 50000) return '10-50KB';
+  if (size <= 100000) return '50-100KB';
+  if (size <= 200000) return '100-200KB';
+  if (size <= 500000) return '200-500KB';
+  return '>500KB';
+}
+
+export const AnalyseArgs = {
+  ...logArguments,
+  path: positional({ type: Url, displayName: 'path', description: 'Path to mbtiles' }),
```

Comment:
I think we don't support multiple mbitles? Might just covert to a option parameter?